certsuite/harness.py

```diff
@@ -451,6 +453,8 @@ def run_tests(args, config):
     finally:
         if output_zipfile:
             print >> sys.stderr, "Results saved to %s" % output_zipfile
+        if remove_after_mcts:
+            marionette_install.uninstall()
```

Comment:
I've lost my familiarity with this code. Would you mind pointing me to the source code of the uninstall function?